docs/get-started/tokenization/security_token.md

```diff
@@ -117,6 +117,15 @@ Creating security tokens is the same as creating NFTs or FTs, with the extra con
 		unitName, assetName, assetURL, assetMetadataHash)
     ```
 
+To give some additional background information, here's a short explanation of each of the properties:
+
+- Manager address: The manager account is the only account that can authorize transactions to [re-configure](https://developer.algorand.org/docs/get-details/asa/#modifying-an-asset) or [destroy](https://developer.algorand.org/docs/get-details/asa/#destroying-an-asset) an asset.
+- Reserve address: Specifying a reserve account signifies that non-minted assets will reside in that account instead of the default creator account. Assets transferred from this account are "minted" units of the asset. If you specify a new reserve address, you must make sure the new account has opted into the asset and then issue a transaction to transfer all assets to the new reserve.
+- Freeze address: The freeze account is allowed to freeze or unfreeze the asset holdings for a specific account. When an account is frozen it cannot send or receive the frozen asset.
```

Comment:
The account _may_ send these tokens back to the creator account. I believe it can also receive more tokens, but please double check me

Reply:
from docs "An Asset Freeze Transaction is issued by the Freeze Address and results in the asset receiver address losing or being granted the ability to send or receive the frozen asset."

No info in docs if they can send tokens back to the creator account when frozen?

Reply:
Yes, I just learned you can do this a well. It should be documented there as well. If you're up for it feel free otherwise I can do it. It may need to be an asset close to transaction, I'm not sure. it's the only type I tried

Reply:
Updated, feel free to modify to make it more accurate.